olp-cpp-sdk-core/src/cache/DiskCache.cpp

```diff
@@ -445,5 +409,32 @@ bool DiskCache::RemoveKeysWithPrefix(const std::string& prefix,
   return result.IsSuccessful();
 }
 
+void DiskCache::InitializeDB(const StorageSettings& settings,
+                             const std::string& path) const {
+  std::unique_ptr<leveldb::DB> database;
+  leveldb::DB* db = nullptr;
+
+  auto open_options = CreateOpenOptions(settings, false);
+  leveldb::DB::Open(open_options, path, &db);
```

Comment:
Why did you drop the status? Even if the first attempt is erroneous and we attempt to repair it doesn't mean that the second try will succeed. I don't think we should drop the status.